easybuild/easyblocks/t/trinity.py

```diff
@@ -279,7 +286,16 @@ def install_step(self):
             else:
                 raise EasyBuildError("Don't know how to set TRINITY_COMPILER for %s compiler", comp_fam)
 
-            cmd = "make TRINITY_COMPILER=%s" % trinity_compiler
+            explicit_make_args = ''
+            if LooseVersion(self.version) > "2" and LooseVersion(self.version) < "3":
+                explicit_make_args = 'all plugins'
+            # Delete CXXFLAGS CPPFLAGS and CFLAGS (interfere with plugins installation)
+            if LooseVersion(self.version) > "2" and LooseVersion(self.version) < "3":
+                del os.environ['CXXFLAGS']
+                del os.environ['CPPFLAGS']
+                del os.environ['CFLAGS']
```

Comment:
@boegel: Yes, I am a bit afraid of this. I am not 100% sure how we have to handle this, as it seems that the `Trinity` authors recommend only `Chrysalis` and `Inchworm` to be compiled with Intel, and the rest with `GCC`. I do not know what is the correct solution in this cases, as I do not know how to pull the right `GCC` compiler flags. (And I am not sure that we do not have to use the hard-coded options). Anyway, one of our user already got into a problem with `Intel` compiled `blastx`, it gave wrong results (but not `blastn`), and apparently the `foss` version was rather similar in speed .